gips/0080-gip-direct-indexing-payments.md

```diff
@@ -0,0 +1,147 @@
+
+The following is a sequence diagram
+
+![image](https://github.com/user-attachments/assets/1182cc24-9adf-4aea-969a-954b9364389f)
+
+### Post-MVP Contracts-Based Implementation
+The Post-MVP phase depends on the Subgraph Service implementation from Graph Horizon, and uses a trust-minimized payment mechanism for Indexers to collect payment from an Escrow (that is the same Escrow used by TAP for query fee payments). 
+
+The Gateway commits to paying the specified price with a voucher, that is posted on-chain by the Indexer, and that the contracts will use as a pre-authorization. 
+
+This allows the Indexer to collect payment directly when posting the POI, and having a guarantee that the payment will be available when the job is done (as long as the Escrow is sufficiently funded). 
+
+At this phase, we will likely also iterate on the pricing approach and introduce a better "indexing gas" metric as an evolution to the MVP entities-and-blocks-based approach. 
+
+### Product vs Protocol
+The GIP is primarily enabling Consumers, who most likely will be Gateways, to interact via the protocol with the Indexer Network.
+
+However, the actual Products which are the interface for Developers and End Customers, need a way to successfully build their business ontop of the Protocol.
+
+This illustration shows the relationship between the two:
+
+![image](https://github.com/user-attachments/assets/0415f01d-4b8f-4bf0-9579-72acbe037a51)
+
+### What is a Product?
+
+The Product Experience **defines the UX for the End User**, in most cases, a dapp developer who wants data that has been indexed.
+
+A good Product experience includes as much **predictability and transparency upfront for potential costs to the paying Developer.**
+
+However, there could be instances the Gateway has to take on risks for the sake of simplicity.
+
+**This decision is a Product decision** made by the Studio operator.
+
+### What is the role of the Gateway in the Product experience?
+
+The Studio operator will often be running the Gateway, as well (although if a trusted experience is established with a third-party Studio, this could also be possible).
+
+The Gateway, then, is like the payment gateway **and** the Indexing Indexer Selection Algorithm (IISA) for the Studio operator.  As a payment gateway, the Gateway operates more like Stripe: a utility facilitating the payment flow via escrow.  However, the IISA **could** be considered part of the product experience, a means for the Studio to potentially differentiate in the marketplace.
+
+Could another Studio use the same Gateway?
+
+If the Studio used an existing Studio via an API, such as a Studio-as-a-Service”, then another Studio would inherit the trust assumptions of the Gateway operator (since the Studio and Gateway are the same operator). 
```

Comment:
This sentence looks a bit confusing - the studio using an existing studio? imo it's not necessary to go into so much detail about things on the Product side, in the end GIPs are more about changes to the Protocol, and I think we only need to mention that there will be UX improvements that are possible on the Product side but won't directly affect the protocol design